File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Main Key
    int n, i, b, s;
    do
    {
        n = get_int("positive number: ");
    }
    while (n < 1 || n > 8);

    //pyramid row
    for (i = 0; i < n; i++)
    {
        //spacebar to change alignment
        for (s = 0; s < n - i - 1; s++)
        {
            printf(" ");
        }

        //pyramid column
        for (b = 0; b <= i; b++)
        {
            //print brick
            printf("#");
        }
        //move to next row
        printf("\n");
    }
}

File: sentimental-mario-less/mario.py
from cs50 import get_int

# promptuser for positive number
while True:
    n = get_int('height: ')
    if n >= 1 and n <= 8:
        break

# print row and column
for i in range(n):
    for c in range(n):
        if i + c >= n - 1:
            print('#', end='')
        else:
            print(' ', end='')
    print()